README.md

```diff
@@ -13,360 +13,176 @@
-- As of now, only HTTPS calls to `github.com`, `api.github.com`, `*.pkg.github.com`, and `ghcr.io` hosts are monitoried.
+Harden-Runner offers a comprehensive suite of features to enhance the security of your CI/CD workflows, available in two tiers: **Community** (Free) and **Enterprise** (Paid).
 
-<p align="left">
-  <img src="images/https-events.png" alt="View outbound HTTPS traffic at the job level" >
-</p>
+### Community (Free)
 
-### Detect anomalous outbound network traffic
+- **Block Network Egress Traffic with Domain Allowlist:** Control outbound network traffic by specifying allowed domains, preventing unauthorized data exfiltration.
+- **Detect Compromised Packages, Dependencies & Build Tools:** Identify and mitigate risks from malicious or vulnerable components in your build process.
+- **Detect Modification of Source Code:** Monitor and alert on unauthorized changes to your source code during the CI/CD pipeline.
+- **Disable Sudo Access:** Restrict the use of superuser privileges in your workflows to minimize security risks.
+- **Insights Page for CI/CD Runs:** Access detailed reports and analytics for each CI/CD run to monitor security events and compliance.
 
-> Applies to both GitHub-hosted and self-hosted runners
+### Enterprise (Paid)
 
-You can detect suspicious/ anomalous traffic using this feature even in `egress-policy:audit` mode.
+Includes all features in the **Community** tier, plus:
 
-- Anomaly detection feature creates a machine learning model of outbound network calls by analyzing the historical data of the same workflow in previous runs
-- After the baseline is created, any anomalous outbound destinations are marked as anomalous in the insights page, and real-time alerts are triggered
-- You can view the list of all anomalous outbound network traffic in the `Runtime detections` page on the dashboard
+- **Support for Private Repositories:** Extend Harden-Runner's security capabilities to your private GitHub repositories.
+- **Support for Self-Hosted Runners:** Apply security controls and monitoring to self-hosted GitHub Actions runners.
+- **View Outbound GitHub API calls at the Job Level:** Monitor HTTPS requests to GitHub APIs
+- **Determine Minimum GITHUB_TOKEN Permissions:** Monitor outbound HTTPS requests to GitHub APIs to recommend the least-privilege permissions needed for your workflows, enhancing security by reducing unnecessary access.
+- **View the Name and Path of Every File Written During the Build Process:** Gain visibility into every file written to the build environment, including the ability to correlate file writes with processes, ensuring complete transparency.
+- **View Process Names and Arguments:** Monitor every process executed during the build process, along with its arguments, and navigate the process tree to detect suspicious activities.
 
```

Comment:
We added a new feature `GitHub Checks` which can also be added here. More details can be found at
https://int.docs.stepsecurity.io/harden-runner/how-tos/github-checks/